### PR TITLE
OSD: Add icon for Slow-Mo

### LINKED ISSUE #10805: [UI Oversight/Bug]: Slo Mo uses the fast forward icon (like limiter off)
### Describe the Bug

Going in to slo mo mode uses the double arrows, like limiter off, this is a little confusing from a UI perspective, we should probably have some sort of different icon for that.

The universal one seems to be

image

However this won't fit our UI so maybe something like ..|> which will give a similar sentiment

### Reproduction Steps

Load a game/bios, press Shift + Tab

### Expected Behavior

_No response_

### PCSX2 Revision

Any up to 1.7.5553

### Operating System

Windows 10 (64bit)

### If Linux - Specify Distro

_No response_

### Logs & Dumps

_No response_

### PR DESCRIPTION
**Pull request by kamfretoz**
### Description of Changes

This PR adds dedicated status indicator for Slow-Mo and properly links the indicator to their appropriate speed.

Slow-Mo:
Screenshot_20240212_210423

200% Speed:
Screenshot_20240212_210419

Turbo:
Screenshot_20240212_210405

Fixes #10805 

### Rationale behind Changes

Confusion 0

### Suggested Testing Steps

Test said functionality, swap between all 3 speed limiters.

### CODE REVIEW

File: pcsx2/ImGui/ImGuiOverlays.cpp [hunk 1]
```diff
@@ -241,10 +242,14 @@ __ri void ImGuiManager::DrawPerformanceOverlay(float& position_y, float scale, f
...
-			{
-				const bool is_slowmo = (target_speed < EmuConfig.EmulationSpeed.NominalScalar);
-				DRAW_LINE(standard_font, is_slowmo ? ICON_FA_FORWARD : ICON_FA_FAST_FORWARD, IM_COL32(255, 255, 255, 255));
-			}
+			  {
+			      if (target_speed == EmuConfig.EmulationSpeed.SlomoScalar) //	Slow-Motion
+			          DRAW_LINE(standard_font, ICON_PF_SLOW_MOTION, IM_COL32(255, 255, 255, 255));
+			      else if (target_speed == EmuConfig.EmulationSpeed.TurboScalar)  // Turbo (Fast-Forward)
+			          DRAW_LINE(standard_font, ICON_FA_FAST_FORWARD, IM_COL32(255, 255, 255, 255));
+			      else // 200% Speed
```
Comment (refractionpcsx2) [inline pcsx2/ImGui/ImGuiOverlays.cpp:250]: comment should be `// Unlimited`

Comment (kamfretoz) [reply pcsx2/ImGui/ImGuiOverlays.cpp:250]: Fixed

Comment (refractionpcsx2) [reply pcsx2/ImGui/ImGuiOverlays.cpp:250]: No you haven't?

Comment (refractionpcsx2) [reply pcsx2/ImGui/ImGuiOverlays.cpp:250]: oh also can you fix the indenting please

Comment (kamfretoz) [reply pcsx2/ImGui/ImGuiOverlays.cpp:250]: oh shoot

Comment (kamfretoz) [reply pcsx2/ImGui/ImGuiOverlays.cpp]: Fixed for realsies this time

**Pull request merged**